### PR TITLE
spell: comply with new checks

### PR DESCRIPTION
**Pull request by apahim**
make spell is now covering more cases. This patch fixes the actual spell
errors and adds the required exceptions.

Signed-off-by: Amador Pahim <apahim@redhat.com>

### CODE REVIEW

**Comment by apahim** [comment]
[BUG: make spell broken](https://trello.com/c/Ibvuwl5V/1195-bug-make-spell-broken)

File: spell.ignore [hunk 1]
```diff
@@ -453,3 +453,99 @@ dm
...
+lucas
+meneghel
+rodrigues
+lukas
+doktor
+ruda
+moura
+amador
+pahim
+rosa
```
Comment (ldoktor) [inline spell.ignore]: nice section :-) (did you consider adding `# authors` section perhaps on top of this file?)

Comment (ldoktor) [reply spell.ignore]: (I mean that is not a request, it's just a finishing touch which is not really needed)

File: spell.ignore [hunk 2]
```diff
@@ -453,3 +453,99 @@ dm
...
+travis
+testclass
+extensionmanager
+jobpre
+jobpost
+joberror
+admanski
+basename
+bligh
+andy
```
Comment (ldoktor) [inline spell.ignore]: those are also authors ;-)

File: spell.ignore (hunk 1 above)
Comment (apahim) [reply spell.ignore]: No, I didn't consider, but now that you mentioned, sounds like a good touch. I will send a V2 including that. Thanks.

Comment (ldoktor) [reply spell.ignore]: As we discussed let's keep it this way

**Pull request merged**